apollo-gradle-plugin/build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 plugins {
   id("org.jetbrains.kotlin.jvm")
-  id("apollo.library")
   id("java-gradle-plugin")
+  id("apollo.library")
```

Comment:
"apollo.library" needs to be after "java-gradle-plugin" or it creates 2 overlapping maven publications